core/storage/file/gae_models.py

```diff
@@ -49,30 +49,88 @@ def get_new_id(cls, entity_name):
 
     @classmethod
     def get_undeleted(cls):
+        """Retrieve values that have not been deleted in the file metadata
+        model.
+
+        Returns:
+            list. A list of undeleted items from the query.
```

Comment:
This should be `list(something).`, where the 'something' is the type of an element of the list. E.g. list(str) or list(int) or list(FileMetadataModel) ...